src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1OutOfServiceBedDetailsEntity.kt

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1OutOfServiceBedHistoryType
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.EnumSet
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Repository
+interface Cas1OutOfServiceBedDetailsRepository : JpaRepository<Cas1OutOfServiceBedDetailsEntity, UUID>
+
+@Entity
+@Table(name = "cas1_out_of_service_bed_details")
+data class Cas1OutOfServiceBedDetailsEntity(
```

Comment:
Would it be clearer if this were named in way which made it clear that the entity represents a change or an update to an OOSB entity? e.g.

- `Cas1OutOfServiceBedChangeEntity` or
- `Cas1OutOfServiceBedUpdateEntity`